webapp/channels/src/components/admin_console/remove_file_setting.tsx

```diff
@@ -1,71 +1,71 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {PureComponent} from 'react';
-import type {MouseEvent} from 'react';
+import React, {memo, useCallback, useState} from 'react';
+import type {FC, MouseEvent} from 'react';
 
 import Setting from './setting';
 import type {Props as SettingsProps} from './setting';
 
 type Props = SettingsProps & {
     id: string;
     label: React.ReactNode;
-    helptext?: React.ReactNode;
+    helpText?: React.ReactNode;
     removeButtonText: React.ReactNode;
     removingText?: React.ReactNode;
     fileName: string;
     onSubmit: (arg0: string, arg1: () => void) => void;
     disabled?: boolean;
 }
 
-type State = {
-    removing: boolean;
-}
-
-export default class RemoveFileSetting extends PureComponent<Props, State> {
-    constructor(props: Props) {
-        super(props);
-
-        this.state = {
-            removing: false,
-        };
-    }
+const RemoveFileSetting: FC<Props> = (props) => {
+    const {
+        id,
+        label,
+        helpText,
+        removeButtonText,
+        removingText,
+        fileName,
+        onSubmit,
+        disabled,
+    } = props;
```

Comment:
I would change it this way:
```suggestion
const RemoveFileSetting = ({
        id,
        label,
        helpText,
        removeButtonText,
        removingText,
        fileName,
        onSubmit,
        disabled,
    }: Props) => {
```

I did it in the editor, so do not commit it directly, because can be broken.

Reply:
👍🏻